script/generate-workflow-files

```diff
@@ -54,7 +54,8 @@ def write_test_workflow(flavors):
                     'uses': 'docker/build-push-action@v4',
                     'with': {
                         'context': 'all',
-                        'load': True,
+                        # 'load': True,
```

Comment:
If this is going to stick around commented out it probably makes sense to add a comment as to why.

Reply:
Removed!